### PR TITLE
Fixes #73

### LINKED ISSUE #73: Analysis: Edits
- [ ] - Consistent reference to "Analyst" i.e. decide on case, and singular or plural
- [x] - 8.2 " The analyst will also carry out the assurance plan" -> " The analyst will also carry out the Assurance Plan, and conduct verification and validation as detailed in the Assurance Plan"
- [x] - 8.2 Unclear what this means: "The log should detail the source, ownership and a fitness-for-purpose risk assessment;"
- [x] - 8.3 "The assurer may then need to re-review the assurance completed, as required." -> "The assurer may then need to re-review the analytical work completed, as required."
- [x] - 8.4 "The commissioner’s role is to review the analysis and assurance as fit for the purposes of the commission." -> "The commissioner’s role is to review any changes in design or methodology that the Analyst brings to their attention. The Commissioner should be available to provide input and clarifications to the Analyst"
- [x] - 8.5 "The Approver scrutinises the work of the analyst and assurer and shall be confident that the analysis:
- [x]       Meets its design requirements;Is of sufficient quality and Is adequately documented." Is this in Analysis or in sign-off?
- [x] - 8.6 Can we make this refer specifically to the Analysis phase? Current: "It is not sufficient to identify and describe risks and uncertainties. It is essential also to assess their impact on the outcome of the analysis and their contribution to the range and likelihoods of possible outcomes. Guidance on how to identify, mitigate, measure and communicate uncertainty can be found in the [Uncertainty Toolkit for Analysts](https://analystsuncertaintytoolkit.github.io/UncertaintyWeb/index.html)." -> "While the Scoping and Design phases identified and described risks and uncertainties, the Analysis phase aims to assess and quantify the impact of uncertainty on the analytical outcome and their contribution to the range and likelihoods of possible outcomes. Guidance on how to identify, mitigate, measure and communicate uncertainty can be found in the [Uncertainty Toolkit for Analysts](https://analystsuncertaintytoolkit.github.io/UncertaintyWeb/index.html)."
- [x] Multi-use models in Analysis phase
- [x] AI/ML in Analysis phase

### PR DESCRIPTION
**Pull request by irisoren-sg**
Also changed subheadings and levels.

@Hurstharrier  I have added several new issues as questions while reviewing this section (#131 - #134). Please review the current branch and merge if you approve, and we can deal with the new issues as a separate pull request

### CODE REVIEW

File: analysis.qmd [hunk 1]
```diff
@@ -6,48 +6,59 @@ The draft currently has no official status. It is a work in progress and is subj
...
-In larger analytical projects, some outputs of the analysis may be completed at different times as work progresses, and aspects of other phases may therefore take place concurrently.
 
-## The Analyst's responsibilities during the analysis phase
+
+### The Analyst's responsibilities during the analysis phase
 
 As the analyst manages their analysis and follows their analytical plan, they must provide documentation of the data and methods used to the assurers. The analyst shall ensure these are sufficient for the assurer to understand the approach.
 
-When the analysts are following the analytical plan, they must record any changes to their plan if they encounter difficulties or unexpected limitations. The analyst will also carry out the assurance plan and shall provide traceable documentation of the assurance they have undertaken. They shall act on any findings from the assurer. 
+The analyst must record any changes to the analytical plan if they encounter difficulties or unexpected limitations.
```
Comment (Hurstharrier) [inline analysis.qmd:20]: Think the analysis recording changes in the plan should be a "should".

The analyst shall record changes to the analytical plan if they encounter difficulties or unexpected limitations. The final version of the plan shall reflect the analysis carried out. The aim being to enable the analysis to be repeated.

File: analysis.qmd [hunk 2]
```diff
@@ -6,48 +6,59 @@ The draft currently has no official status. It is a work in progress and is subj
...
-## The Analyst's responsibilities during the analysis phase
+
+### The Analyst's responsibilities during the analysis phase
 
 As the analyst manages their analysis and follows their analytical plan, they must provide documentation of the data and methods used to the assurers. The analyst shall ensure these are sufficient for the assurer to understand the approach.
 
-When the analysts are following the analytical plan, they must record any changes to their plan if they encounter difficulties or unexpected limitations. The analyst will also carry out the assurance plan and shall provide traceable documentation of the assurance they have undertaken. They shall act on any findings from the assurer. 
+The analyst must record any changes to the analytical plan if they encounter difficulties or unexpected limitations. 
+
+The analyst will also follow the assurance plan, and conduct the specified verification and validation. They shall provide traceable documentation of the assurance they have undertaken. They shall act on any findings from the assurer.
```
Comment (Hurstharrier) [inline analysis.qmd:22]: needs to be "shall follow the assurance plan"

File: analysis.qmd [hunk 3]
```diff
@@ -6,48 +6,59 @@ The draft currently has no official status. It is a work in progress and is subj
...
-## The Approver's responsibilities during the analysis phase
+While the Scoping and Design phases identified and described risks and uncertainties, the Analysis phase aims to assess and quantify the impact of uncertainty on the analytical outcome and their contribution to the range and likelihoods of possible outcomes. Guidance on how to identify, mitigate, measure and communicate uncertainty can be found in the [Uncertainty Toolkit for Analysts](https://analystsuncertaintytoolkit.github.io/UncertaintyWeb/index.html). The verification and validation by the analyst and assurer should assure the appropriate treatment of uncertainty.
 
-The Approver scrutinises the work of the analyst and assurer and shall be confident that the analysis:
+## Multi-use models and the analysis phase
 
-* Meets its design requirements;
-* Is of sufficient quality and
-* Is adequately documented.
+In multi-use models, analysis and edits may be carried out on individual elements of the model at differing times. This calls for mechanisms for assuring that the changes integrate into the larger model as expected, for example, through the use of test-suites.
```
Comment (Hurstharrier) [inline analysis.qmd:57]: On multi use. I agree about the point on change control. 

Think that the role of the commissioner and approver should be restated and say that this function is normally done by a committee.

File: analysis.qmd [hunk 4]
```diff
@@ -6,48 +6,62 @@ The draft currently has no official status. It is a work in progress and is subj
...
-The Approver scrutinises the work of the analyst and assurer and shall be confident that the analysis:
+## Artificial Intellingence and Machine Learning (Black Box) models and the analysis phase
 
-* Meets its design requirements;
-* Is of sufficient quality and
-* Is adequately documented.
+AI and ML models are not as transparent as traditionally coded models. This adds challenge to the assurance of these models as compared to other forms of analysis. Assurance activities during this phase include performance testing and formal verification. See the [Introduction to AI Assurance](https://www.gov.uk/government/publications/introduction-to-ai-assurance) for further details.
 
-## Uncertainty
+## Working with third parties
```
Comment (Hurstharrier) [inline analysis.qmd:63]: On working with third parties - academics, contractors etc. Think we need to define what we mean by a third party in the definitions sector. The bit we haven't covered is the act of commissioning the work. We have a Commissioner who is basically the customer and then what normally happens is an analysis speicfies the work for a thrid party. Thus there is an intermeidary. 
Seems to me the important pointhere is that the specification is clear about who performs each role i.e. is the third party acting as the Analyst or the Analyst and Assurer or...
Do we need a  name for the person specifying the work  for the third party?

**Comment by Hurstharrier** [comment]
@irisoren-sg  have reviewed and left comments. 

The bit on third parties prompted some thoughts for the whole book. I will start a discussion.

**Review by Hurstharrier — COMMENTED** [review]
@irisoren-sg  I have reviewed and left comments. 

The bit on third parties has prompted some further thoughts that may need other changes in the rest of the book. I will start a discussion

**Comment by zarbspace** [comment]
This all looks sensible to me, happy to close off.

**Pull request merged**